teams/roles-and-responsibilities.md

```diff
@@ -1 +1,65 @@
-# Teams: Roles and Responsibilities
+# Ember Core Team Member Roles, Responsibilities, and Expectations
+
+## Participation
+
+As a bare minimum, core team members are expected to:
+
+- Follow the community guidelines- https://emberjs.com/guidelines 
+- Interact thoughtfully:
+    - with the community in an exemplary way - for better or for worse, the “core team” is often perceived as a unit, not as individuals
+    - with other members of the core teams in a respectful way (do we need to say this?)
+- Lead the official projects through some combination of
+    - Triaging issues
+    - Reviewing PRs and mentoring community contributors
+    - Answering community questions (Discord, [Discuss](https://discuss.emberjs.com/), Stack Overflow, etc)
+    - Supporting community documentation efforts
+    - Writing blog posts for emberjs.com
+    - Championing RFCs
+    - Supporting the release process
+- Attend weekly team meetings
```

Comment:
Not all teams have weekly meetings

Reply:
All core teams do, though.

Reply:
Do we want this to be a mandatory aspect of core teams?

Reply:
@MelSumner - Disagree.

@wycats - No, I don't think so 🤔

Reply:
@wycats yes, I think it should be. If we all share common guidelines and language for those guidelines, we will be able to get closer to achieving the goal of making all core teams peers.